gammapy/scripts/image_pipe.py

```diff
@@ -101,6 +102,36 @@ def bkg_map(self, bkg_norm=True):
 
         self.maps["bkg"] = bkg_map
 
+    def make_1d_exposure(self, spectral_index=2.3, for_integral_flux=False):
+        """Compute the 1D exposure table for one observation for an offset table
+
+        Parameters
+        ----------
+        spectral_index : float
+            Assumed power-law spectral index
+        for_integral_flux : bool
+            True if you want that the total excess / exposure gives the integrated flux
+
+        Returns
+        -------
+
```

Comment:
Document what this `make_1d_exposure` method returns.

Is it really an exposure? Exposure computation doesn't need an assumed spectrum, right?
So maybe it's exposure (= area x time) times flux, i.e. expected counts?
Maybe pick a better name?